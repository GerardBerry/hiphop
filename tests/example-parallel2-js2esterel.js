var par = require("./example-parallel2.js");
var js2esterel = require("../js2esterel.js");

console.log(par.par.esterel_code());
