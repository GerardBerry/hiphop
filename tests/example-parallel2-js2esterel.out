module parallel2:
output J;
signal I in
        present I then
            emit J
        end present
    ||
        emit I
end signal
end module
